### ISSUE TITLE
Will no longer be updated to the Raku ecosystem

### ISSUE THREAD

**Issue by lizmat** [awwaiid/p6-Inline-Ruby#6]
Hello,

Thank you for the time and effort you have put in what is now the [Raku Programming Language](https://raku.org).  The language and its ecosystem are [moving forward](https://github.com/Raku/problem-solving/pull/318).

This means that these distributions:

- https://github.com/awwaiid/p6-Inline-Ruby
- https://github.com/awwaiid/p6-lrep

will no longer be checked for updates by the Raku ecosystem.

This means that you will need to move your module to the [zef ecosystem](https://deathbykeystroke.com/articles/20210120-faq-zef-ecosystem.html) on the next update you make to these distributions.  Otherwise your update will simply not be seen by the ecosystem.

If you are no longer interested in maintaining these distributions, you can have these distributions moved to [Raku Community Modules Adoption Center](https://github.com/raku-community-modules/), where members of the Raku Community will take care of them.

This can be as easy as confirming that in a response to this issue.

Again, thank you for your time and efforts!

On behalf of the Raku Steering Council,

Elizabeth Mattijsen

**Comment by awwaiid** [comment]
Yes, these distributions are open for adoption

**Comment by lizmat** [comment]
Could you please transfer these distributions to the raku-community-modules project?  You should have sufficient rights for that.  I will then take care of them.

**Comment by awwaiid** [comment]
Hi! I transferred Inline::Ruby! Sad but also that doesn't stop me from maybe coming around and contributing! :)

**Comment by lizmat** [comment]
Indeed, it won't!   And thanks!

**Comment by lizmat** [comment]
In light of recent [`REPL`](https://raku.land/zef:lizmat/REPL) developments it feels correct if you would archive the lrep repo.

**Comment by awwaiid** [comment]
@lizmat yes I agree -- for LREP I'll add a note and a link and archive. Again I can always un-archive if I suddenly make progress, but I'm assuming LREP depends on some internal details that may have changed in the last 8 years hah.
